src/rpc/net.cpp

```diff
@@ -426,7 +429,6 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
             "  \"protocolversion\": xxxxx,              (numeric) the protocol version\n"
             "  \"localservices\": \"xxxxxxxxxxxxxxxx\", (string) the services we offer to the network\n"
             "  \"localrelay\": true|false,              (bool) true if transaction relay is requested from peers\n"
-            "  \"timeoffset\": xxxxx,                   (numeric) the time offset\n"
```

Comment:
It doesn't seem to have been removed in Bitcoin.

Reply:
Yes, this is a little different now. Earlier, each node maintains its own time offset from to the median block time. So this information was available globally and reported in network info. Now it is maintained by the node for each peer and reported only in peerinfo rpc.